types/chromecast-api/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for chromecast-api 0.4
+// Project: https://github.com/alxhotel/chromecast-api
+// Definitions by: SteffoSpieler <https://github.com/SteffoSpieler>
+//                 daniel0611 <https://github.com/daniel0611>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import Client from "./lib/client";
```

Comment:
```suggestion
import Client = require("./lib/client");
```
IMO this is all CJS module, old school:
https://github.com/alxhotel/chromecast-api/blob/master/index.js